pod/perlvar.pod

```diff
@@ -885,7 +885,8 @@ command or referenced as a file.
 
 =back
 
-=head2 Variables related to regular expressions
+=head2 Regex Variables
```

Comment:
I don't see the need for changing this heading when it would break any existing links to this section.

Reply:
I couldn't find any. Do you mean those outside of the core?

Reply:
In general, any documentation may link to sections in core docs

Reply:
I'd like to rename it. See latest version of the patch.